Assets/Scripts/Model/Types/Player.cs

```diff
@@ -5,28 +5,47 @@ namespace Fumbbl.Model.Types
 {
     public class Player : ViewObject<Player>
     {
-        public string Id;
-        public string PositionId;
-        public string Name;
-        public string PortraitURL;
-        public Types.Coordinate Coordinate { get; set; }
+        public Player()
```

Comment:
As a general rule, fields and properties before constructor. I know the DTOs have the constructor first and if you insist on unifying this, it's better to change the DTOs to match the more general case.